modules/config/base.go

```
@@ -1,141 +1,135 @@
-		os.Exit(1)
-	}
-	return dev, debug, port, tiny, workdirectory, flushTokenTime, databasepath, maxsaved, historyEnable, sampletime, netdata_routernum
-}
 
-func checkErr(err error) {
-	if err != nil {
-		panic(err)
+	// Check for updates
+	autocheckupdatebool, _ := strconv.ParseBool(autocheckupdate)
+	if !Cfg.Tiny {
+		download.DownloadStatic(Cfg.Workdirectory, false, autocheckupdatebool)
```

golangci-lint (1.64.8)

Error return value of `download.DownloadStatic` is not checked

(errcheck)